stock_available_immediately/product.py

```diff
@@ -55,7 +53,7 @@ def _product_available(self, cr, uid, ids, field_names=None,
         if 'immediately_usable_qty' in field_names:
             for product_id, stock_qty in res.iteritems():
                 res[product_id]['immediately_usable_qty'] = \
-                    stock_qty['qty_available'] + stock_qty['outgoing_qty']
+                    stock_qty['qty_available'] - stock_qty['outgoing_qty']
```

Comment:
In the newest v7 this was changed to available immediately = available - incomming.
I hope it's ok for you?